Gemfile

```diff
@@ -2,6 +2,7 @@ source 'https://rubygems.org'
 
 gem "activesupport", ">= 5.2.7", :require => false
 gem "bootstrap-sass", "~> 3.x"
+gem "dotenv"
```

Comment:
Should we set `MIQ_ENV=production` instead? https://github.com/ManageIQ/manageiq.org/blob/e814021fbc4e467d76fa4f2c9b1de8152fb811b4/lib/miq.rb#L4-L9
We don't have a .env file in production and I don't see any other references to this gem

Reply:
Done